lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -25,6 +25,7 @@ export class PublicClientApplication extends ClientApplication implements IPubli
 
     // Active requests
     private activeSilentTokenRequests: Map<string, Promise<AuthenticationResult>>;
+    private astsAsyncMeasurement?: InProgressPerformanceEvent = undefined;
```

Comment:
What did the asts stand for?  @tnorling this seems to be from an older change.

Reply:
Not sure...probably a typo and was supposed to be "ATS" for acquireTokenSilent?

Reply:
@peterzenz Thanks for pointing that out. Will create a separate PR for pre-existing typos.

Reply:
fixed